git_machete/github.py

```diff
@@ -6,24 +6,32 @@
 import shutil
 import subprocess
 # Deliberately NOT using much more convenient `requests` to avoid external dependencies
-from http.client import HTTPResponse, HTTPSConnection
+from urllib.request import Request, urlopen
+from urllib.error import HTTPError
+import http
```

Comment:
Nit: keep imports in abc order (probably https://github.com/VirtusLab/git-machete/issues/140 will keep them sorted anyway)